armory/__main__.py

```diff
@@ -668,6 +668,8 @@ def launch(command_args, prog, description):
     (config, args) = arguments.merge_config_and_args(config, args)
 
     rig = Evaluator(config, root=args.root)
+    if not args.interactive and not args.jupyter:
```

Comment:
In a common case case, `args.jupyter` is `False`  so this conditional degrades to `args.interactive = True`. That is, it forces interactive mode regardless of what was put on the command line.  Is this the intent?

Reply:
Yes, that's exactly the stated intent.